webapp/tests/karma/ts/services/tasks-for-contact.service.spec.ts

```diff
@@ -71,7 +77,7 @@ describe('TasksForContact service', function() {
         type: contactType,
       };
 
-      const tasks = await service(model);
+      const tasks = await service.get(model);
       assert.equal(fetchTaskDocsFor.callCount, 0);
```

Comment:
I wonder why original dev used `assert` and then `expect` instead of sticking with one. I would keep 1